react-function-components/src/index.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const CustomButton = () => <button>Click Me!</button>;
+
+ReactDOM.render(
+  CustomButton(),
```

Comment:
In react we never really call our own components, instead, we create react elements with them.

```jsx
ReactDOM.render(
  <CustomButton />,
  document.querySelector('#root')
);
```